keyboards/littlealby/mute/config.h

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2019 albybarber
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+#include "config_common.h"
+
+/* USB Device descriptor parameter - https://github.com/qmk/qmk_firmware/blob/master/docs/config_options.md#hardware-options */
+#define VENDOR_ID       0x5342 // "SB"
+#define PRODUCT_ID      0x3225 // "2%"
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Little Alby
+#define PRODUCT         Mute
+
+/* USB_MAX_POWER_CONSUMPTION - sets the maximum power (in mA) over USB for the device (default: 500)*/
```

Comment:
```suggestion
```

Reply:
This is a helpful comment, why do you want to remove it?

Reply:
The docs already explain what this does.

Reply:
Ok, fair point. You win this round 😁